pkg/api/universal/conversation.go

```diff
@@ -51,9 +53,29 @@ func (a *Universal) ConverseAlpha1(ctx context.Context, req *runtimev1pb.Convers
 		return nil, err
 	}
 
+	scrubber, err := piiscrubber.NewDefaultScrubber()
```

Comment:
Do we want to be caching `scrubber`, or creating it on `universal.New()`. ?

Reply:
It's an extremely cheap pass-through method that can't ever return an error, so I'm fine keeping it as is